ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh

```diff
@@ -82,4 +81,4 @@ LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
 LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
   --zone "${GOOGLE_COMPUTE_ZONE}" \
   packer@"${INSTANCE_PREFIX}" \
-  --command "export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}  && timeout 360m bash -ce \"/home/packer/run-tests.sh\""
+  --command "timeout 360m bash -ce \"/home/packer/run-tests.sh\""
```

Comment:
This commit looks good to me, we'll be using the default release from `snc.sh` rather than using a different one forced by the CI environment.